npm/build-sandbox-types.js

```diff
@@ -129,6 +129,20 @@ module.exports = function (exit) {
                             c.type.typeName.escapedText = `import("postman-collection").${currentType}`;
                         }
                     }
+
+                    // For properties referencing sdk and some more properties, eg. request: Request|IRequest
+                    if (c.type && c.type.types && c.type.types.length) {
```

Comment:
This will happen if there is `c.types.types.types`. Shouldn't this be recursive?